test/integration/avo/rubygems_test.rb

```diff
@@ -17,7 +17,7 @@ class Avo::RubygemsTest < ActionDispatch::IntegrationTest
     assert_response :success
     assert page.has_content? rubygem.name
 
-    get avo.resources_rubygem_path(rubygem)
+    get avo.resources_rubygem_path(rubygem.slug)
```

Comment:
If I understand it well, this is not needed, since in admin we use `id`.

Reply:
```suggestion
    get avo.resources_rubygem_path(rubygem)
```